src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -110,6 +113,21 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
       )
   }
 
+  lazy private[overwatch] val verboseAuditModule = Module(3018, "Gold_VerboseAuditLog", this, Array(1004,3004,3005))
+  lazy private val appendVerboseAuditProccess: () => ETLDefinition = {
+    () =>
+      ETLDefinition(
+        BronzeTargets.auditLogsTarget.asIncrementalDF(verboseAuditModule, BronzeTargets.auditLogsTarget.incrementalColumns),
+        Seq(buildVerboseAudit(
+          GoldTargets.notebookTarget,
+          GoldTargets.clusterStateFactTarget
```

Comment:
this must be incremental DFs

CLSF should have lag of 30d to ensure long-running states are within scope but not forever history